core/project/Dependencies.scala

```diff
@@ -8,8 +8,9 @@ object Version {
   val AkkaMgmt      = "1.0.8"
   val AkkaGrpc      = "1.0.1"
   val AlpakkaKafka  = "2.0.5"
-  val Scala         = "2.12.11"
-  val ScalaOperator = "2.13.3"
+  val Scala212      = "2.12.11"
+  val Scala213      = "2.13.3"
```

Comment:
I renamed them to Scala212 and Scala213 so we clear see which version is behind.